### PR TITLE
Fixed Python Supervisor API documentation

### PR DESCRIPTION
**Pull request by omichel**
Fixed small mismatch between the Supervisor Python API and its documentation.

### CODE REVIEW

File: docs/reference/supervisor.md [hunk 1]
```diff
@@ -1991,7 +1991,7 @@ namespace webots {
 from controller import Supervisor
 
 class Supervisor (Robot):
-    def setLabel(self, id, label, xpos, ypos, size, color, transparency, font="Arial"):
+    def setLabel(self, id, label, x, y, size, color, transparency, font="Arial"):
```
Comment (ad-daniel) [inline docs/reference/supervisor.md:1994]:
```suggestion
    def setLabel(self, id, label, x, y, size, color, transparency=0, font="Arial"):
```

**Review by ad-daniel — CHANGES_REQUESTED** [review]
Thank you

**Review by ad-daniel — APPROVED** [review]

File: docs/reference/supervisor.md [hunk 2]
```diff
@@ -1991,7 +1991,7 @@ namespace webots {
 from controller import Supervisor
 
 class Supervisor (Robot):
-    def setLabel(self, id, label, xpos, ypos, size, color, transparency, font="Arial"):
+    def setLabel(self, id, label, x, y, size, color, transparency=0, font='Arial'):
```
Comment (stefaniapedrazzi) [inline docs/reference/supervisor.md:1994]: Why the transparency has a default value in the Python API but not in the C++ API?

Comment (omichel) [reply docs/reference/supervisor.md:1994]: I agree, this is inconsistent. I will add this default argument to the C++ and Java APIs...

**Pull request merged**